Simplified/StringUtilities.m

```diff
@@ -0,0 +1,26 @@
+//
+//  StringUtilities.m
+//  Simplified
+//
+//  Created by Aferdita Muriqi on 6/20/17.
+//  Copyright © 2017 NYPL Labs. All rights reserved.
+//
+
+#import "StringUtilities.h"
+
+@implementation StringUtils
+
+NSString * NYPLLocalizedString(NSString * key, NSString * comment) {
```

Comment:
This is a minor style point, but I would attach the `*` to the identifier it modifies:

```objective-c
NSString *NYPLLocalizedString(NSString *key, NSString *comment)
```

This convention is used throughout the project as far as I know. The reason for this convention is that the `*` modifies the name, not the type. For example, the following declaration declares `x` as a pointer to an integer yet it declares `y` merely as an integer:

```c
int *x, y;
```